packages/pieces/stripe/src/lib/trigger/new-customer.ts

```diff
@@ -42,15 +42,15 @@ export const stripeNewCustomer = createTrigger({
   },
   type: TriggerStrategy.WEBHOOK,
   async onEnable(context) {
-    const webhook = await stripeCommon.subscribeWebhook('customer.created', context.webhookUrl, context.auth);
+    const webhook = await stripeCommon.subscribeWebhook('customer.created', context.webhookUrl, context.auth.publicKey);
     await context.store.put<WebhookInformation>('_new_customer_trigger', {
       webhookId: webhook.id
     });
   },
   async onDisable(context) {
     const response = await context.store?.get<WebhookInformation>('_new_customer_trigger');
     if (response !== null && response !== undefined) {
-      await stripeCommon.unsubscribeWebhook(response.webhookId, context.auth);
+      await stripeCommon.unsubscribeWebhook(response.webhookId, context.auth.publicKey);
```

Comment:
We should revert auth to secret key, because I think subscribe webhooks uses secrets key as well because they register endpoint.